### DISCUSSION TITLE
Why does npm fail 'Exit status 1' at the end of the test run, if there is a single test that fails assert

### DISCUSSION THREAD

**Discussion by ElmaCat** [getgauge/taiko#2051] [Development]
When I run the test suite and all tests execute correctly. I set up one tests intentionally to fail the assert as expected.
In the console log I can see all test results but after the results I also see the error. When I remove the failing scenario, the npm error does not appear.
![image](https://user-images.githubusercontent.com/17129392/117702392-58684e00-b1c0-11eb-87f9-989eca5063c2.png)

Gauge version: 1.1.8
"taiko": "1.2.2"

The pipeline is also failing because of this. 
![image](https://user-images.githubusercontent.com/17129392/117703160-53f06500-b1c1-11eb-9b1c-485e7711d3b2.png)

**Comment by NivedhaSenthil** [comment]
That is the default behavior of npm, it propagates the exit code
